packages/css-library/tokens/color/base.json

```diff
@@ -0,0 +1,36 @@
+{
+  "color": {
+    "base": { "value": "#212121"},
+    "white": { "value": "#ffffff"},
+    "black": { "value": "#000000"},
+    "orange": { "value": "#eb7f29"},
+    "link-default": { "value": "#004795"},
+    "warning-message": { "value": "#fac922"},
+    "gibill-accent": { "value": "#fff1d2"},
+    "primary": { "value": "#0071bb"},
+    "primary-darker": { "value": "#003e73"},
+    "primary-darkest": { "value": "#112e51"},
+    "primary-alt": { "value": "#02bfe7"},
+    "primary-alt-dark": { "value": "#00a6d2"},
+    "primary-alt-darkest": { "value": "#046b99"},
+    "primary-alt-light": { "value": "#9bdaf1"},
+    "primary-alt-lightest": { "value": "#e1f3f8"},
+    "secondary": { "value": "#e31c3d"},
+    "secondary-dark": { "value": "#cd2026"},
+    "secondary-darkest": { "value": "#981b1e"},
+    "secondary-light": { "value": "#e59393"},
+    "secondary-lightest": { "value": "#f9dede"},
+    "gray": {
+      "gray": { "name": "color-gray", "value": "#5b616b"},
+      "dark": { "value": "#323a45"},
```

Comment:
I left the primary & secondary groups of colors un-nested just to show some different options for structuring the tokens. This line right here would give us a `color-gray-dark` token.